Pilot/pilotCommands.py

```diff
@@ -456,20 +538,29 @@ def _getBasicsCFG(self):
         if self.pp.gateway:
             self.cfg.append('-W "%s"' % self.pp.gateway)
         if self.pp.userGroup:
-            self.cfg.append('-o /AgentJobRequirements/OwnerGroup="%s"' % self.pp.userGroup)
+            self.cfg.append(
+                '-o /AgentJobRequirements/OwnerGroup="%s"' % self.pp.userGroup
+            )
         if self.pp.userDN:
             self.cfg.append('-o /AgentJobRequirements/OwnerDN="%s"' % self.pp.userDN)
         self.cfg.append("-o /LocalSite/ReleaseVersion=%s" % self.releaseVersion)
 
         if self.pp.wnVO:
-            self.cfg.append('-o "/Resources/Computing/CEDefaults/VirtualOrganization=%s"' % self.pp.wnVO)
+            self.cfg.append(
+                '-o "/Resources/Computing/CEDefaults/VirtualOrganization=%s"'
+                % self.pp.wnVO
+            )
 
     def _getSecurityCFG(self):
         """Nothing specific by default, but need to know host cert and key location in case they are needed"""
         if self.pp.useServerCertificate:
             self.cfg.append("--UseServerCertificate")
-            self.cfg.append("-o /DIRAC/Security/CertFile=%s/hostcert.pem" % self.pp.certsLocation)
-            self.cfg.append("-o /DIRAC/Security/KeyFile=%s/hostkey.pem" % self.pp.certsLocation)
+            self.cfg.append(
+                "-o /DIRAC/Security/CertFile=%s/hostcert.pem" % self.pp.certsLocation
+            )
+            self.cfg.append(
+                "-o /DIRAC/Security/KeyFile=%s/hostkey.pem" % self.pp.certsLocation
+            )
```

Comment:
Interesting....
I have pre-commit installed. Sometimes black fails which I find annoying because it is difficult to verify what black changed. I installed black into my editor (PyCharm) and it blacks on every save, so I can easily see the changes. Then when I commit black is run again and does not complain, so I assume the style changes are compatible. Am I missing something ?

Reply:
I guessed it was like that. The issue I have is that all these changes are mixed in making review rather annoying. What is the version of black that you use with pycharm? And does it have some settings?

Reply:
I believe I pip installed black  with my anaconda python3.9 (see below)
It is executed (as far as I understand PyCharm with a filename only. I would need to add --config option I believe. Where is it ? I cannot see it in my repo.

(base) martynia@Januszs-Mac-mini Pilot % black --version
black, 22.6.0 (compiled: yes)
Python (CPython) 3.9.7
(base) martynia@Januszs-Mac-mini Pilot % python
Python 3.9.7 (default, Sep 16 2021, 08:50:36) 
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.

Reply:
Actually black does not have any config file. And we have the same version. 

So, I guess that simply we need to accept these formats.

Reply:
Could we get a similar config file in pilot repositories ?

Reply:
> Could we get a similar config file in pilot repositories ?

yes: https://github.com/DIRACGrid/Pilot/pull/163